src/group_theory/coset.lean

```diff
@@ -181,51 +181,67 @@ theorem normal_of_eq_cosets (h : ∀ g : α, g *l s = s *r g) : s.normal :=
 theorem normal_iff_eq_cosets : s.normal ↔ ∀ g : α, g *l s = s *r g :=
 ⟨@eq_cosets_of_normal _ _ s, normal_of_eq_cosets s⟩
 
+@[to_additive left_add_coset_eq_iff]
+lemma left_coset_eq_iff {x y : α} : left_coset x s = left_coset y s ↔ x⁻¹ * y ∈ s :=
```

Comment:
In principle I could have written this
```suggestion
lemma left_coset_eq_iff {x y : α} : left_coset_equivalence s x y ↔ x⁻¹ * y ∈ s :=
```
but the `left_coset_equivalence` definition seems pretty useless and the user would just have to unfold it.